modules/deploy/partials/cloud/controlplane-api.adoc

```diff
@@ -52,12 +60,17 @@ curl https://api.redpanda.com/v1beta2/operations/<operation-id>
 
 The response contains the current state of the operation: `IN_PROGRESS`, `COMPLETED`, or `FAILED`.
 
+ifndef::env-serverless[]
+
 == Cluster tiers
 
 When you create a BYOC or Dedicated cluster, you select a usage tier. Each tier provides tested and guaranteed workload configurations for throughput, logical partitions, and connections. Availability depends on the region and the cluster type. See the full list of regions, zones, and tiers available with each provider in the xref:api:ROOT:cloud-api.adoc#api-description[API reference].
 
+endif::[]
+
 == Create a cluster
 
+ifndef::env-serverless[]
 To create a new cluster, first create a resource group and network, if you have not already done so.
 
 === Create a resource group 
```

Comment:
Serverless comes with a default resource group created. Please make this step optional. Mention, that users can use the resource group provided.